hardhat/README.md

```diff
@@ -0,0 +1,60 @@
+# Klaytn ServiceChain Token Bridge with Hardhat
+
+This project ports service chain token test/deploy to hardhat.
+
+## Testing on Hardhat network
+Run `npx hardhat run scripts/deploy.js`.
```

Comment:
How about adding instructions to install dependencies by running `npm install`?

Reply:
done